roles/installer/ospd/overcloud/scale/tasks/main.yaml

```diff
@@ -0,0 +1,36 @@
+- name: Set scaler variables
+  set_fact:
```

Comment:
no need to use "set_fact", please read how to sent a variable to a role in the next comment